src/components/ClassSchedule.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { FC } from "react";
+import { ClassBlockProps } from "@/lib/types";
+import ClassBlock from "./ClassBlock";
+
+type ClassScheduleProps = {
+  schedule: ClassBlockProps[];
+  day: string;
+};
```

Comment:
to może być interfacem, moja praktyka jest taka, że dopóki da się użyć interface'u to go właśnie używam